scripts/vote_3_step_3[svn]deploy_artifacts.sh

```diff
@@ -14,7 +14,8 @@ echo "RC ${ignite_version}${rc_name}"
 # Uncomment subsequent line in case you want to remove incorrectly prepared RC
 #svn rm -m "Removing redundant Release" https://dist.apache.org/repos/dist/dev/ignite/$ignite_version$rc_name || true
 svn import svn/vote https://dist.apache.org/repos/dist/dev/ignite/$ignite_version$rc_name -m "New RC ${ignite_version}${rc_name}: Binaries"
-svn import packaging/pkg https://dist.apache.org/repos/dist/dev/ignite/packages_$ignite_version$rc_name -m "New RC ${ignite_version}${rc_name}: Packages"
+
+svn import packages/pkg https://dist.apache.org/repos/dist/dev/ignite/packages_$ignite_version$rc_name -m "New RC ${ignite_version}${rc_name}: Packages"
```

Comment:
Guess we should stop uploading packages to SVN.

Reply:
We can stop uploading (BTW, I didn't upload packages before 2.7.5 votes, nobody mentioned it), 

But since this code is already here, I prefer to keep it, for now. Later, we can discuss changes on the dev.list.

Reply:
We should comment it out then.

Reply:
For now, I prefer to keep uploading working as-is, because we hadn't discussed it with all the community.